jwst/extract_1d/extract.py

```diff
@@ -2089,10 +2087,12 @@ def extract(self, data, wl_array, verbose):
         n_nan = nan_mask.sum(dtype=np.intp)
         if n_nan > 0:
             if verbose:
-                log.warning("%d NaNs in wavelength array", n_nan)
-            temp_wl[nan_mask] = 0.01            # because NaNs cause problems
+                log.debug("%d NaNs in wavelength array", n_nan)
```

Comment:
Not needed for this PR, but it would be nice to get rid of `verbose` and just do logging, allowing (eventually) the user to set the level, in order to get the log level one wants.  But we need to fix `stpipe` logging first.

Reply:
This is a little obscure (maybe more than a little), but the reason for a `verbose` argument in some of the functions in `extract_1d` is to allow logging info for just the first integration of a multi-integration exposure.  For TSO data, in particular, there can be zillions of integrations, and we don't need to log the same info for every one.  So `verbose` is set to True for the first integration, and then it's reset to False.

Reply:
Ah, that makes sense.  Maybe we should _add_ `verbose` to the `ami` code.  😂